### PR TITLE
Add Mithril

### PR DESCRIPTION
**Pull request by porsager**
Add [Mithril.js](https://mithril.js.org)

### CODE REVIEW

**Comment by robdodson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: libraries/mithril/meta/summary.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<h4 id="mithril-handling-data">Handling data</h4>
+
+Mithril will pass data to an element as properties, as long as the property
```
Comment (robdodson) [inline libraries/mithril/meta/summary.md]: Based on the conversation in https://github.com/MithrilJS/mithril.js/issues/2220 it seems like this isn't true yet for arrays and objects?

Comment (porsager) [reply libraries/mithril/meta/summary.md]: Ah, very good point. I've fixed it, and I'll make a new pull request once https://github.com/MithrilJS/mithril.js/pull/2221 lands.

**Comment by robdodson** [comment]
Awesome!!! Thank you for taking the time to put this together!

**Comment by porsager** [comment]
You're welcome, and thank you for the site ;)

**Pull request merged**